src/libraries/System.Buffers/README.md

```diff
@@ -0,0 +1,15 @@
+# System.Buffers
+Contains types used in creating and managing memory buffers, such as those represented by `Span<T>` and `Memory<T>`.
+
+Documentation can be found here: https://learn.microsoft.com/en-us/dotnet/api/system.buffers.
+
+## Contribution Bar
+- [x] [We consider new features, new APIs and performance changes](../../libraries/README.md#primary-bar)
+
+See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-System.Buffers+label%3A%22help+wanted%22+) issues.
+
+## Source
+* [../../coreclr/System.Private.CoreLib/src/System](../../coreclr/System.Private.CoreLib/src/System)
```

Comment:
For source within System.Private.CoreLib the tests are often (always?) in different locations (or the tests remain where they were, but the implementation got moved lower into SPCLib).

So the place where to find the tests isn't always easy to find. Therefore it would be ideal to have some info about this here, e.g. with a separate section about tests.